examples/segmentation/example_supervoxels.cpp

```diff
@@ -230,8 +230,10 @@ main (int argc, char ** argv)
           new_point.z = depth;
         }
         
-        uint32_t rgb = static_cast<uint32_t>(color_pixel[0]) << 16 |  static_cast<uint32_t>(color_pixel[1]) << 8 |  static_cast<uint32_t>(color_pixel[2]);
```

Comment:
Sorry for being picky, but this is not needed. Default constructor will properly initialize Alpha to 255.

Reply:
Be as picky as you need. Everyone benefits in the end.  

I pushed on a separate commit, because it's just easier to see the changes and because we can squash on github.